zulip_bots/zulip_bots/bots/joinme/joinme.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+import requests
+import json
+import socket
+from six.moves.configparser import SafeConfigParser
```

Comment:
We don't support Python 2, so we don't need to use six. From Python 3.2 on, SafeConfigParser is available as ConfigParser. So you can just use 
```.py
from configparser import ConfigParser
```